e2e/README.md

```diff
@@ -196,6 +196,16 @@ class ExampleInput extends React.Component {
 
 It is recommended to follow the scheme parentID/ChildDescription.
 
+## Setting remote config defaults for e2e tests
+
+We do not read remote config values from firebase for e2e tests, so default remote config values are used. A custom set of remote config
+defaults may be needed for some tests which differs from production configuration. For example, in production code,
+we do not wish to show FiatConnect cash-outs by default, but in order to test FiatConnect cash-outs end-to-end, we need
```

Comment:
```suggestion
we do not wish to show FiatConnect cash-outs by default, but to test FiatConnect cash-outs end-to-end, we need
```

Reply:
in the future @MuckT can you leave docs suggestions like this^ so I can accept them with a click?